crates/pico_macros/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "pico_macros"
+version.workspace = true
+edition.workspace = true
+license.workspace = true
```

Comment:
nit: we need some way (not necessarily in this PR) to start publishing crates in CI